### PR TITLE
WAZO-1560 user import: allow the line_protocol to be webrtc

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

File: integration_tests/suite/base/test_user_import.py [hunk 1]
```diff
@@ -361,6 +361,26 @@ def test_given_csv_has_all_sip_fields_then_sip_endpoint_created():
...
+    sip_id = get_import_field(response, 'sip_id')
+    line_id = get_import_field(response, 'line_id')
+
+    sip = confd.endpoints.sip(sip_id).get().item
+    assert_that(
+        sip,
+        has_entries(
+            id=sip_id,
+            line=has_entries(id=line_id),
+            options=has_items(contains('webrtc', 'yes')),
```
Comment (bloom1) [inline integration_tests/suite/base/test_user_import.py]: Wouldn't it be best to test for all options required for WebRTC?

Comment (pc-m) [reply integration_tests/suite/base/test_user_import.py]: Fixed

File: wazo_confd/plugins/user_import/creators.py [hunk 2]
```diff
@@ -163,6 +164,26 @@ def create(self, fields, tenant_uuid):
         return self.service.create(SIP(tenant_uuid=tenant_uuid, **form))
 
 
+class WebRTCCreator(SipCreator):
+    def create(self, fields, tenant_uuid):
+        #  TODO(pc-m): This will need to get changed to "transport-wss" when PJSIP transport gets merged
+        fields.update(
+            {
+                'options': [
+                    ['transport', 'wss'],
```
Comment (pc-m) [inline wazo_confd/plugins/user_import/creators.py]: This line conflicts with the PJSIP transport branch and should be changed to `transport-wss`

Comment (bloom1) [reply wazo_confd/plugins/user_import/creators.py]: > This line conflicts with the PJSIP transport branch and should be changed to `transport-wss`

I will wait for the PJSIP transport to be merged before merging then :+1:

Comment (pc-m) [reply wazo_confd/plugins/user_import/creators.py]: PJSIP transports is merged. This branch has been rebased.

**Pull request merged**